usine.py

```diff
@@ -302,10 +305,13 @@ def _call_command(self, cmd, **kwargs):
                      channel.recv_exit_status())
         channel.close()
         if ret.code:
-            print(red(ret.stderr))
-            sys.exit(ret.code)
+            self.exit(ret.stderr, ret.code)
         return ret
 
+    def exit(self, msg, code=1):
```

Comment:
I would have named that one `error` given that we only want to `exit` on errors?

Reply:
You mean naming the function or the `msg` parameter?

Reply:
The naming

Reply:
Humm, I'm not sure to agree then ;)
I think `exit` explains more what the function does: it *exits* the script (in the same meaning as `sys.exit`).

Reply:
Alright, go for it then 👍